### PR TITLE
1825/feat/contact-questioning-styles

### PR DESCRIPTION
**Pull request by Noam6421**
Look at ticket: [1825](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1825) to see the changes made in the UI.

Screenshots of before & after: 
all

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx [hunk 1]
```diff
@@ -172,32 +165,34 @@ const ContactQuestioningCheck: React.FC<Props> = (props: Props): JSX.Element =>
...
-                                        }
-                                    </Select>
-                                </FormControl>)
+                                return (
+                                    <FormControl variant='outlined' fullWidth>
+                                        <Select
+                                            {...props}
+                                            placeholder={ContactQuestioningFieldsNames.OCCUPATION}
+                                            onChange={(event) => {
+                                                props.onChange(event.target.value)
```
Comment (y-golde) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningCheck.tsx:182]: Can you use event.target?.value ? If possible(With the question mark) , Im not certain but I remember once it crashed the client on a bug I worked on

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx [hunk 2]
```diff
@@ -111,50 +114,52 @@ const ContactQuestioningClinical: React.FC<Props> = (props: Props): JSX.Element
...
-                                    </FormControl>)
+                                    return (
+                                        <FormControl variant='outlined' fullWidth>
+                                            <Select
+                                                {...props}
+                                                disabled={isFieldDisabled || isFamilyContact}
+                                                test-id='familyRelationshipSelect'
+                                                placeholder='קרבה משפחתית'
+                                                onChange={(event) => {
+                                                    props.onChange(event.target.value)
```
Comment (y-golde) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx:140]: Same here, maybe it's more worthwhile to try and break it to see what can happen

**Review by y-golde — APPROVED** [review]
Nice work :)

**Review by chenFriedman — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx (hunk 2 above)
Comment (Noam6421) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx:140]: I tried, it seems to work good :)

**Pull request merged**